digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/middleware/DataImportManager.scala

```diff
@@ -4,6 +4,8 @@ import java.io.InputStream
 import fi.liikennevirasto.digiroad2.asset._
 import fi.liikennevirasto.digiroad2.service.RoadLinkService
 import fi.liikennevirasto.digiroad2._
+import fi.liikennevirasto.digiroad2.client.vvh.VVHClient
+import fi.liikennevirasto.digiroad2.csvDataImporter.{MaintenanceRoadCsvImporter, MassTransitStopCsvImporter, MassTransitStopCsvOperation, ObstaclesCsvImporter, PedestrianCrossingCsvImporter, RailwayCrossingCsvImporter, RoadLinkCsvImporter, ServicePointCsvImporter, TrafficLightsCsvImporter, TrafficSignCsvImporter}
```

Comment:
I think it's better to change this to fi.liikennevirasto.digiroad2.csvDataImporter._